app/layouts/default-layout.tsx

```diff
@@ -196,6 +202,7 @@ export default createFragmentContainer(DefaultLayout, {
     fragment defaultLayout_session on JwtToken {
       ciipUserBySub {
         __typename
```

Comment:
You don't need `__typename` anymore. Its only purpose is when you don't actually need any field in a graphql node, but just want to check its existence. Now that you have a fragment in this node you can remove it.
```suggestion
```

Reply:
Cool thanks for mentioning what it was doing!